internal/pkg/cli/secret_init.go

```diff
@@ -438,13 +473,24 @@ func (o *secretInitOpts) targetEnv(envName string) (*config.Environment, error)
 	return env, nil
 }
 
-// BuildSecretInitCmd build the command for creating a new secret or updating an existing one.
-func BuildSecretInitCmd() *cobra.Command {
+// buildSecretInitCmd build the command for creating a new secret or updating an existing one.
+func buildSecretInitCmd() *cobra.Command {
 	vars := secretInitVars{}
 	cmd := &cobra.Command{
-		Use:     "secret init",
-		Short:   "Create or update an SSM SecureString parameter.",
-		Example: ``, // TODO
+		Use:   "init",
+		Short: "Create or update secrets in SSM Parameter Store.",
+		Example: `
+Create a secret by following our guided experience. 
+You will be prompted for the secret's name and the values you want for it in each existing environments.
+/code $ copilot secret init
+Create a secret named db-password. 
+You will be prompted for the values you want for db-password in each existing environments.
+/code $ copilot secret init --name db-password
+Create a secret named db-password with values you want in each existing environments. 
+You will be prompted for the values you want for db-password in each existing environments.
+/code $ copilot secret init --name db-password
```

Comment:
Is this repeated twice?

Reply:
Addressed! I also addressed your feedback from last round (sorry I missed your comment somehow), shortening the example description language so that they are more terse. However I removed the one with `--values` flag since we want to encourage the user to use guided/input file so that the values are not visible in shell history.